skills/declarative/calendarSkill/runtime/customaction/Actions/MSGraph/GetContacts.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Net.Http;
+using System.Text.RegularExpressions;
```

Comment:
nit: if we order the using's alphabetically we can eliminate a lot of the compiler warnings + make the code easier to read.

Reply:
Yes, we need to add in the stylecop ruleset to these projects. I'll add that as a separate work item

Reply:
Actually our build already generates warnings for these types of issue. At least that's what I see when we are compiling main.

Reply:
Fixed this line. I think there must be lots of style warning in this project. We can fix all at once later separately.